src/libumf.c

```diff
@@ -50,12 +60,26 @@ void umfTearDown(void) {
         umfDestroyTopology();
 #endif
         umfIpcCacheGlobalTearDown();
+
+        if (utils_is_running_in_proxy_lib_with_size_threshold()) {
+            // We cannot destroy the TRACKER nor the base allocator
+            // when we are running in the proxy library with a size threshold,
+            // because it could lead to calling the system free() with an invalid pointer
```

Comment:
I meant "_It could lead to **something**_", so "_calling ..._" , not "_call ..._"

Reply:
I believe `lead to` should be followed by either **something** or `to <verb>`
you could write it somehow differently e.g. `it could result in calling`, or `lead to system free call with ...`

that's my understanding, but you know... I'm not native ;)

I'd re-phrase with `it could result in calling ... and  a segfault.`

Reply:
Replaced with `it could result in calling ...`
Done.

Reply:
now the last part `as a result` is redundant.

Reply:
Done